teamone/settings.py

```diff
@@ -177,8 +186,8 @@
 )
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = "M1y9_Z-jNwqArncabT7_zZWV"
 
-AWS_ACCESS_KEY_ID = "AKIA46VH6MQWLLV5ADGA"
-AWS_SECRET_ACCESS_KEY = "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"
+AWS_ACCESS_KEY_ID = "AKIA46VH6MQWFGIZCO5M"
+AWS_SECRET_ACCESS_KEY = "CDYn206+Etf/tHBOBi6gqQOeZsHI616SNDNELN31"
```

Comment:
@ahanif16 I already put these changes in my recent commit. Can you discard your changes to the AWS key locally and then pull from develop one more time to get the updated AWS key from my commit? Otherwise there will be a merge conflict